pkg/controller/common/helpers_test.go

```diff
@@ -380,3 +381,35 @@ func TestRemoveIgnDuplicateFilesAndUnits(t *testing.T) {
 
 	assert.Equal(t, expectedIgn2Config, convertedIgn2Config)
 }
+
+func TestDiffMachineConfigFilesForPool(t *testing.T) {
```

Comment:
Would `TestCalculateConfigFileDiffs` be more conventional?

Reply:
Indeed it would!